### PR TITLE
Update to alpha08

### PR DESCRIPTION
**Pull request by pfmaggi**
This update the sample to [alpha08](https://developer.android.com/jetpack/androidx/releases/window#1.0.0-alpha08):

- update library version
- add sample usage for `WindowInfoRepoJavaAdapter`
- Update README file to new coroutine/flow syntax introduced in [alpha07](https://developer.android.com/jetpack/androidx/releases/window#1.0.0-alpha07)

### CODE REVIEW

File: WindowManager/app/src/main/java/com/example/windowmanagersample/SplitLayoutActivity.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import android.os.Bundle;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.util.Consumer;
+import androidx.window.WindowInfoRepo;
+import androidx.window.WindowLayoutInfo;
+import androidx.window.java.WindowInfoRepoJavaAdapter;
+import com.example.windowmanagersample.databinding.ActivitySplitLayoutBinding;
+
+public class SplitLayoutActivity extends AppCompatActivity {
```
Comment (kford55) [inline WindowManager/app/src/main/java/com/example/windowmanagersample/SplitLayoutActivity.java:28]: Is there a reason we swapped this back to java for the sample? Just to showcase the JavaAdapter?

Comment (pfmaggi) [reply WindowManager/app/src/main/java/com/example/windowmanagersample/SplitLayoutActivity.java:28]: Yes, the idea was to showcase how to use the JavaAdapter's API.

File: WindowManager/README.md [hunk 2]
```diff
@@ -19,80 +19,142 @@ full advantage of whatever device they are on. The Jetpack WindowManager
 library allows you to handle all of these devices through a common API as well
 as through different versions of Android.
 
-You can determine what `DisplayFeatures`s are available on the device and their
-`Rect` location. The alpha02 release introduces a new `DisplayFeature` class with an updated callback contract to notify your application when a `DisplayFeature` changes. You can register/unregister the callback using these methods:
+You can determine what [`DisplayFeature`][0]'s are available on the device
```
[the remark this answers is not in the document]
Comment (pfmaggi) [reply WindowManager/README.md:22]: Markdown allows to use an id for links. I usually use this as I find it easier to read the source doc.
I can revert to having the link in the text if you think that it is what we usually use.

ref: https://daringfireball.net/projects/markdown/syntax#link

[the remark this answers is not in the document]
Comment (kford55) [reply WindowManager/README.md:22]: Didn't know this was a thing! :+1:

**Comment by kford55** [comment]
:+1:

**Review by kford55 — APPROVED** [review]
:+1:

**Pull request merged**